rules/windows/privilege_escalation_uac_bypass_event_viewer.toml

```diff
@@ -14,7 +14,67 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Bypass UAC via Event Viewer"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Bypass UAC via Event Viewer
+
+Windows User Account Control (UAC) allows a program to elevate its privileges (tracked as integrity levels ranging from
+low to high) to perform a task under administrator-level permissions, possibly by prompting the user for confirmation.
+The impact to the user ranges from denying the operation under high enforcement to allowing the user to perform the
```

Comment:
Same question for tis second sentence . It's fairly long and can prob be separated into two sentences.

Reply:
Yep, https://attack.mitre.org/techniques/T1548/002/